pkg/types/utils.go

```diff
@@ -32,3 +39,19 @@ func MachineNetworksToCIDRs(nets []MachineNetworkEntry) []configv1.CIDR {
 
 	return res
 }
+
+// GetClusterProfileName utility method to retrieve the cluster profile setting.  This is used
+//
+//	when dealing with openshift api to get FeatureSets.
```

Comment:
nit: comment formatting is a little funky

Reply:
Oh that is odd.  Good catch.